odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -119,6 +127,53 @@ message AddJobSpecificationsResponse {
   string log = 2;
 }
 
+message JobInspectRequest {
+  string project_name = 1;
+  string namespace_name = 2;
+  string job_name = 3;
+  JobSpecification spec = 4;
+}
+
+message BasicInfoSection {
```

Comment:
it would be better to keep all the dependent  messages with in the JobInspectResponse @sbchaos what do you say? Want to keep this very scoped as they won't be shared at all.

Reply:
done